pkg/cortex/cortex_test.go

```diff
@@ -184,6 +184,43 @@ func TestConfigValidation(t *testing.T) {
 			},
 			expectedError: nil,
 		},
+		// NameValidationScheme tests 
+		{
+			name: "should not fail validation for empty name validation scheme (use legacy by default)",
+			getTestConfig: func() *Config {
+				configuration := newDefaultConfig()
+				configuration.NameValidationScheme = ""
+				return configuration
+			},
+			expectedError: nil,
+		},
+		{
+			name: "should not fail validation for legacy name validation scheme",
+			getTestConfig: func() *Config {
+				configuration := newDefaultConfig()
+				configuration.NameValidationScheme = "legacy"
+				return configuration
+			},
+			expectedError: nil,
+		},
+		{
+			name: "should not fail validation for utf-8 name validation scheme",
+			getTestConfig: func() *Config {
+				configuration := newDefaultConfig()
+				configuration.NameValidationScheme = "utf-8"
+				return configuration
+			},
+			expectedError: nil,
+		},
+		{
+			name: "should fail validation for invalid(anything other than legacy and utf-8) name validation scheme",
```

Comment:
You can just say `should fail validation for invalid name validation scheme`. No need to mention legacy and utf8